src/applications/personalization/view-dependents/components/ViewDependentsHeader/ViewDependentsHeader.jsx

```diff
@@ -10,7 +13,9 @@ const ViewDependentsHeader = () => (
         dependent.
       </p>
       <a
-        href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=dependent-compensation"
+        href={eBenefitsUrl(
+          'ebenefits/about/feature?feature=dependent-compensation',
+        )}
         className="usa-button-primary va-button-primary"
```

Comment:
Following up on my suggestion below, this component would remain a functional component, but the eBenefits link itself would be a container. 

```jsx
     <h1>Your VA Dependents</h1>
      <p className="vads-u-font-size--md vads-u-font-family--serif">
        Below is a list of dependents we have on file for you. You can file a
        claim for additional disability compensation whenever you add a new
        dependent.
      </p>
      <eBenefitsLink 
         feature={ebenefitsFeatures.DEPENDENT_COMPENSATION}
          className="usa-button-primary va-button-primary">
          Add or remove a dependent
      </eBenefitsLink>
```

The `feature` prop could just be wishful thinking if there are more shapes to the eBenefits URLs than just the `?feature=` query flag.  But in that case, you could just pass along the whole `href` -

```jsx
    <eBenefitsLink 
         href="ebenefits/about/feature?feature=dependent-compensation"
          className="usa-button-primary va-button-primary">
          Add or remove a dependent
      </eBenefitsLink>
```